### PR TITLE
index.html: Prefer HTTPS for assets

### PR DESCRIPTION
**Pull request by alanorth**
All assets are hosted on servers configured for HTTPS, so this is absolutely preferable in this day and age. In addition to privacy, there is an argument that there's no point in requesting resources over the Internet if you can't verify where they come from. Functionality is not changed at all.

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -131,14 +131,14 @@ <h5 class="title">Contributing</h5>
 
 			<footer class="footer">
 				<section class="container">
-					<p>Designed with ♥ by <a target="_blank" href="http://cjpatoilo.com" title="CJ Patoilo">CJ Patoilo</a>. Licensed under the <a target="_blank" href="https://github.com/milligram/milligram#license" title="MIT License">MIT License</a>.</p>
+					<p>Designed with ♥ by <a target="_blank" href="https://cjpatoilo.com" title="CJ Patoilo">CJ Patoilo</a>. Licensed under the <a target="_blank" href="https://github.com/milligram/milligram#license" title="MIT License">MIT License</a>.</p>
```
Comment (alkchr) [inline demo/index.html]: GH Pages do not support `https` with custom domains, do they? If only, @cjpatoilo is going to use Cloudflare.

Comment (alanorth) [inline demo/index.html]: Ah! Indeed. Good catch. Shall I `git push -f` to the branch for this pull request or?

**Comment by alanorth** [comment]
Updated pull request to remove HTTPS from author's homepage (`http://cjpatoilo.com`).

**Comment by cjpatoilo** [comment]
Hi @alanorth Thanks for PR.

**Pull request merged**